template/src/my-command.js

```diff
@@ -1,15 +1,15 @@
 import BrowserWindow from 'sketch-module-web-view'
-const UI = require('sketch/ui')
+import UI from 'sketch/ui'
 
-export default function() {
+export default = () => {
   const options = {
     identifier: 'unique.id',
     width: 240,
     height: 180,
     show: false
   }
 
-  var browserWindow = new BrowserWindow(options)
+  const browserWindow = new BrowserWindow(options)
```

Comment:
@mathieudutour Do we need to keep this to `let`? (to remove the reference later on)

Reply:
I think `const` is fine :+1: